provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs

```diff
@@ -219,5 +223,15 @@ private static DeviceRegistrationResult ConvertToProvisioningRegistrationResult(
                 result.Etag,
                 result?.Payload?.ToString(CultureInfo.InvariantCulture));
         }
+
+        private static int jitterMax = 5;
+        private static int jitterMin = 0;
+        private static TimeSpan GenerateDelayWithJitterForRetry(TimeSpan defaultDelay)
```

Comment:
Duplicated code. Please remove/move to common.

Reply:
Sure thing